README.md

```diff
@@ -11,7 +11,7 @@
 
 # Good First Issue Finder by EddieHub
 
-Good First Issue Finder helps new open source contributors pave their path into the world of open source through good first issues.
+Good First Issue Finder helps new open source contributors to make it easy to get into the world of open source through good first issues.
```

Comment:
I think this change is already in main 🤔

Reply:
let me check

Reply:
yes! you are right change is already in main. What should I do now to this PR.